### ISSUE TITLE
Subscriber not called after action dispatch

### ISSUE THREAD

**Issue by perjansson** [amcdnl/ngxs#67]
Component

```
logout() {
	this.ngxs.dispatch(new Logout()).subscribe(() => {
		this.router.navigate(['']); // <-- This code never runs
	});
}
```

Store

```
@Action(Logout)
logout() {
	this.loginService.logout();
}
```

**Comment by deebloo** [comment]
@perjansson thanks for reporting will look into it

**Comment by xmlking** [comment]
I am also doing similar code, I thing you may have to `return` in the action method. 
Reference [here](https://github.com/xmlking/nx-starter-kit/blob/master/libs/auth/src/components/login/login.component.ts#L39)

**Comment by amcdnl** [comment]
I'm seeing this in my 2.0 code too @deebloo ... Its def a bug.

**Comment by deebloo** [comment]
I will investigate the issue and fix in 2.0

**Comment by amcdnl** [comment]
Confirmed fixed in 2.0 branch. Closing this, 2.0 will be released this weekend.
